tests/test_isrTaskLSST.py

```diff
@@ -1069,6 +1069,7 @@ def test_isrFloodedSaturatedE2V(self):
         isr_config.doFlat = False
         # Tun off saturation masking to simulate a PTC flat.
         isr_config.doSaturation = False
+        isr_config.doITLEdgeBleedMask = False
```

Comment:
After seeing all these overrides I wonder if the default should be False?  Though it may be fine but there needs to be a PR for cp_pipe.